src/modules/options/OptionsWidget_servers.cpp

```diff
@@ -1817,8 +1842,8 @@ void OptionsWidget_servers::commit()
 
 void OptionsWidget_servers::customContextMenuRequested(const QPoint &pnt)
```

Comment:
This is a cosmetic thing I noticed when sending PR,
Do you think its worth adding a indenting fix for this function below so all entries are not all over? 

like line 1849, 1860 1862, 1863 etc.